bin/java

```diff
@@ -136,8 +136,10 @@ _get_jdk_download_url() {
     local jdkUrl=$(eval echo \$JDK_URL_1_${jdkVersion})
   elif [ "$(expr "${jdkVersion}" : '^1.[6-9]')" != 0 ]; then
     local jdkUrl="${JDK_BASE_URL}/openjdk${jdkVersion}.tar.gz"
+  elif [ "${jdkVersion}" = "9+181" ] || [ "${jdkVersion}" = "9.0.0" ]; then
+    local jdkUrl="${JDK_BASE_URL}/openjdk9-181.tar.gz"
```

Comment:
Is this intentionally changing "9+181" to "9-181"?

Reply:
I believe the version string is 9+181 but the tarball on s3 are 9-181

Reply:
Yes, the tarball can't have the `+` char in it. When you try to access it as an HTTP URL it 404s.